react/__tests__/Coupon.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render } from '@vtex/test-tools/react'
+import React from 'react'
+import Coupon from '../Coupon'
+import { OrderCouponContext } from 'vtex.order-coupon/OrderCoupon'
+
+const promoButtonText = 'Apply Promo Code'
+
+describe('<Coupon />', () => {
+  const renderComponent = (customProps: any) => {
+    const wrapper = render(
+      <OrderCouponContext.Provider value={customProps}>
+        <Coupon />
+      </OrderCouponContext.Provider>
+    )
+    return wrapper
+  }
+
+  it('should be rendered', () => {
+    const component = renderComponent({})
+    expect(component).toBeDefined()
+  })
+
+  it('should show apply promo buttom', () => {
+    const { getByText } = renderComponent({
+      coupon: '',
+      showPromoButton: true,
+    })
+
+    const element = getByText(promoButtonText)
+
+    expect(element).toBeTruthy()
+    expect(element).toBeDefined()
```

Comment:
This is redundant. If `element` is truthy then it has to be defined (i.e. not `undefined`).

Reply:
done!